src/main/java/org/spongepowered/common/mixin/api/command/SignCommandSourceMixin_API.java

```diff
@@ -34,15 +34,14 @@
 import org.spongepowered.asm.mixin.Final;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.Shadow;
-import org.spongepowered.common.bridge.command.CommandSenderBridge;
 import org.spongepowered.common.command.WrapperCommandSource;
 
-@Mixin(targets = "net/minecraft/tileentity/TileEntitySign$1")
-public abstract class TileEntitySign_1Mixin_API implements SignSource {
+@Mixin(targets = {"net/minecraft/tileentity/TileEntitySign$1", "net/minecraft/tileentity/TileEntitySign$2"})
+public abstract class SignCommandSourceMixin_API implements SignSource {
```

Comment:
@Zidane Would appreciate your guidance on naming this one - how are we naming multi-target mixins? 

Also, I think this shouldn't have moved package as both **targets** are in `tilentity` so they were in the right place.

Reply:
Multi-targets are odd. It is on a case by case basis and basically I name them given a theme (or name them in reference to their bridge).

Also if all the targets exist in one package, leave it in that package.

Reply:
I moved this mixin because it only implements SignSource, thought this was a better place based on its role.

Reply:
It's not so much about its role, more the classes it mixes into. In this case, they are all in the `n.m.tileentity` package, so where it was before is the right place (`o.s.c.m.api.mcp.tileentity`).

The name? Probably `TileEntitySignAnonMixin_API` - it's only targetting anonymous classes in `TileEntitySign`. Alternatively, `SignSourceMixin_API`, naming after the `SignSource` interface you are implementing.

Naming is hard.

Reply:
Since both targets are `ICommandSender` implementations the mixin could be named `TileEntitySignCommandSendersMixin_API`.

I'll move it back to `mcp.tileentity` once a name is decided.

Reply:
That name would work for me.

Reply:
This looks fine to me too.